backend/bang-ggood/src/main/java/com/bang_ggood/article/repository/ArticleRepository.java

```diff
@@ -4,10 +4,28 @@
 import com.bang_ggood.exception.BangggoodException;
 import com.bang_ggood.exception.ExceptionCode;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import java.util.List;
+import java.util.Optional;
 
 public interface ArticleRepository extends JpaRepository<Article, Long> {
 
     default Article getById(Long id) {
         return findById(id).orElseThrow(() -> new BangggoodException(ExceptionCode.ARTICLE_NOT_FOUND));
     }
+
+    @Query("SELECT a FROM Article a WHERE a.id = :id AND a.deleted = false")
+    Optional<Article> findById(@Param("id") Long id);
+
+    @Query("SELECT a FROM Article a WHERE a.deleted = false")
+    List<Article> findAll();
+
+    @Query("SELECT a FROM Article a WHERE a.deleted = false ORDER BY a.createdAt DESC LIMIT 3")
+    List<Article> findLatestTop3();
```

Comment:
limit 개수를 service에서 받아오는 건 어떨까요?? 그리고 Repository 관련 테스트도 필요할 것 같아요!
```suggestion
    @Query("SELECT a FROM Article a WHERE a.deleted = false ORDER BY a.createdAt DESC LIMIT :count")
    List<Article> findLatestTop3(@Param("count") count);
```

Reply:
반영했습니다!